projects/brotli/brotli_fuzzer.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
Please move the fuzzer code in brotli repo, we want fuzz target there. See https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md. Then you you don't need to copy it here.